source/common.h

```diff
@@ -880,8 +880,8 @@ class cmdline_processor
         -> void
     {
         help_requested = true;
-        std::string_view a = __DATE__;
-        std::string_view b = __TIME__;
+        const std::string_view a = __DATE__;
+        const std::string_view b = __TIME__;
```

Comment:
How about:
```suggestion
        constexpr auto a = __DATE__;
        constexpr auto b = __TIME__;
```

Reply:
The `constexpr` I'll add.
`auto` would make for more extensive code changes, will change type to `char *`, which doesn't support `.substr`. Was going for minimal impact changes.